app/src/store/actions.ts

```diff
@@ -307,7 +308,7 @@ export async function searchBusiness ({ getters }, corpNum: string): Promise<Bus
     // first try to find business in Entities (Legal API)
     const data = await NamexServices.searchEntities(corpNum)
     // for restoration requests, verify business eligibility
-    if (getters.isRestoration) {
+    if (getters.isRestoration && data.state !== EntityStates.HISTORICAL) {
```

Comment:
Whoops.

Thanks for the fix! Please follow up with Ethan Portman (Names Team) to deploy this to Test asap. Luckily, this wasn't deployed to Prod.

Reply:
Sure thing, thanks a lot Sev!

Reply:
This is now deployed to Test, and hopefully Prod soon. Names Team will look after it.

Reply:
Yes, thanks Sev. 👍 Ethan already deployed it to Test this morning.